src/sentry/filters/web_crawlers.py

```diff
@@ -1,74 +0,0 @@
-from __future__ import absolute_import
-
-import re
-
-from .base import Filter
-from sentry.utils.data_filters import FilterStatKeys
-from sentry.utils.safe import get_path
-
-# not all of these agents are guaranteed to execute JavaScript, but to avoid
-# overhead of identifying which ones do, and which ones will over time we simply
-# target all of the major ones
-CRAWLERS = re.compile(
-    r'|'.join(
-        (
-            # Google spiders (Adsense and others)
-            # https://support.google.com/webmasters/answer/1061943?hl=en
-            r'Mediapartners\-Google',
```

Comment:
Do all these filters already exist in a default catch-all relay configuration?

Reply:
The regex is part of the web_crawlers filter implementation and one cannot enable / disable it on a per project basis, in other words if you enable web crawlers then all web crawlers contained in  the regex will be filtered. One can only enable or disable the whole filter (this boolean is all that is stored in the relay_config for the web_crawlers filter).